templates/rules.md

```diff
@@ -0,0 +1,31 @@
+# Rules
+
+## TriggerMesh
+
+Please make sure to use capital letters for TriggerMesh known elements like Bridges, Sources, Targets, use `TriggerMesh` and not `Triggermesh`. 
+
+## Unordered Lists (Bullet Lists)
+
+For Unorderd Lists (a.k.a. Bullet Lists) we would like to use of `-` instead of `*`. 
+
+`-` would be the more logical choice, since it eliminates the ambiguity between `*` (bulltet point), `* *` (italic) and `**` `**` (bold). 
+
+   ## Example Bullet List
+
+   - Example 
+   - Example 
+   - Example
+
+## Ordered Lists (Number Lists)
+
+For Ordered Lists (a.k.a. Number Lists) we would like to use `1. 1. 1.` instead of `1. 2. 3.` for listing numbers.
+
+`1. 1. 1.` is actually the recommended way to create numbered lists in Markdown. The generated page will have the correct numbering, not `1. 1. 1.`....
+
+The advantage is that you can add/remove elements to a list without having to renumber everything manually.
+
+   ## Example Number List
```

Comment:
Was the indentation intentional here?

Reply:
@antoineco Yes the indentation was intentional, just wanted to keep the example underneath the rule. If you don't like it I can easily remove the indentation.

Reply:
I'm asking because Markdown ignores this, it just creates another title on the same level as the previous one, not at a sub-level.

If you want a "h3" instead of a "h2" you need to write

```markdown
### Title
```

Reply:
Ok, I will update to `### Title`